packages/venia-ui/lib/components/Button/button.css

```
@@ -1,72 +1,89 @@
 .root {
     composes: root from '../clickable.css';
+    --stroke: var(--venia-brand-color-1-700);
     background: none;
-    border: 1px solid rgb(var(--color));
-    border-radius: 1.5rem;
-    color: rgb(var(--color));
-    font-size: 0.75rem;
-    font-weight: 600;
-    height: 2rem;
-    min-width: 7.5rem;
-    padding: 0 1rem;
-    transition-duration: 384ms;
-    transition-property: background-color, color;
+    border-color: rgb(var(--stroke));
+    border-radius: 10rem;
+    border-style: solid;
+    border-width: 2px;
+    color: rgb(var(--stroke));
+    font-size: var(--venia-global-fontSize-100);
+    font-weight: var(--venia-global-fontWeight-bold);
+    line-height: 1.25rem;
+    max-width: 100%;
+    min-width: 10rem;
+    min-height: 2.5rem;
+    outline: none;
+    padding: calc(0.5rem + 1px) 1.5rem calc(0.5rem - 1px);
+    text-transform: uppercase;
+    transition-duration: 256ms;
+    transition-property: background-color, border-color, color;
     transition-timing-function: var(--venia-global-anim-standard);
-    --color: var(--venia-global-color-text);
-}
-
-.filled {
-    background-color: rgb(var(--color));
-    color: white;
 }
 
 .root:hover {
-    --color: var(--venia-global-color-teal);
+    --stroke: var(--venia-brand-color-1-800);
 }
 
 .root:focus {
-    box-shadow: 0 0 0 2px rgb(var(--venia-global-color-teal-light)),
-        0 0 0.5rem 2px rgba(var(--venia-global-color-teal), 0.2);
-    outline: none;
-    transition-duration: 128ms;
-    --color: var(--venia-global-color-teal);
+    box-shadow: -6px 6px rgba(var(--venia-global-color-blue-700), 0.3);
```

Now that these property values are space-delimited rather than comma-delimited, we need to use the new syntax in which the alpha channel is separated by a slash rather than a comma. This syntax is supported by both `rgb()` and `rgba()`, so we can use the former.

```suggestion
    box-shadow: -6px 6px rgb(var(--venia-global-color-blue-700) / 0.3);
```

---

Updated all instances of `rgba(var` with the new `rob(var(...) /` syntax :)